sdks/ios/runtime/Makefile

```diff
@@ -0,0 +1,72 @@
+all: runtime libmonoios.a
+
+UNREFERENCED_SYMBOLS =                                                  \
+        _xamarin_log                                                            \
+        _xamarin_timezone_get_data                                      \
+        _xamarin_timezone_get_names                                     \
+        _CloseZStream                                                           \
+        _CreateZStream                                                          \
+        _Flush                                                                          \
+        _ReadZStream                                                            \
+        _WriteZStream
+
+CC = /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang
+SIM_SYSROOT = /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk
```

Comment:
this and other places should use the output of `xcode-select -p` instead of `/Applications/Xcode.app`